client/src/components/AuditAdmin/Setup/Settings/Settings.tsx

```diff
@@ -115,7 +190,7 @@ const Settings: React.FC<IProps> = ({
             </FormSection>
             {auditType === 'BALLOT_POLLING' && (
               <FormSection label="Audit Board Data Entry">
-                <label htmlFor="online">
+                <label>
```

Comment:
Resolving a browser warning since this `for` attribute didn't have a corresponding input